doc/plugins/org.polarsys.capella.core.ui.intro/intro/firststeps-doc.xml

```diff
@@ -18,15 +18,15 @@
         path="firststeps/@">
         <group id="content-group" style-id="content-group">
             <link style-id="content-link" label="Visit Capella Website"
-                url="http://org.eclipse.ui.intro/openBrowser?url=http://www.polarsys.org/capella/index.html" id="visitCapellaWebsite">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://www.eclipse.org/capella/index.html" id="visitCapellaWebsite">
                 <text>Read Case-Studies, attend to Upcoming Events, access to Training and Coaching, etc.</text>
             </link>
             <link style-id="content-link" label="Visit Capella Wiki"
-                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.polarsys.org/Capella" id="visitCapellaWiki">
+                url="http://org.eclipse.ui.intro/openBrowser?url=https://wiki.eclipse.org/Capella" id="visitCapellaWiki">
```

Comment:
Ups this one doesn't work either,  its https://github.com/eclipse/capella/wiki

Reply:
done